src/snowflake/snowpark/functions.py

```diff
@@ -3244,7 +3244,7 @@ def hour(e: ColumnOrName) -> Column:
     return builtin("hour")(c)
 
 
-def last_day(e: ColumnOrName) -> Column:
+def last_day(expr: ColumnOrName, part: Optional[ColumnOrName] = "MONTH") -> Column:
```

Comment:
Let's add an `Args` section in the docstring now that there are two parameters.

Reply:
Done!